hubble/pkg/printer/color.go

```diff
@@ -116,3 +118,19 @@ func (c colorer) authTestAlwaysFail(a interface{}) string {
 func (c colorer) authIsEnabled(a interface{}) string {
 	return c.green.Sprint(a)
 }
+
+// compute the list of unique ANSI escape sequences for this colorer.
+func (c *colorer) sequences() []string {
+	unique := make(map[string]struct{})
+	for _, v := range c.colors {
+		seq := v.Sprint("|")
+		split := strings.Split(seq, "|")
+		if len(split) != 2 {
+			// should never happen
+			continue
+		}
+		unique[split[0]] = struct{}{}
+		unique[split[1]] = struct{}{}
+	}
+	return slices.Collect(maps.Keys(unique))
```

Comment:
Ha I should have been more thoughtful about go 1.23+ feature usage.

Should we update this in backport only with:
```
	seqs := make([]string, 0, len(unique))
	for k := range unique {
		seqs = append(seqs, k)
	}
	return seqs
```

Or make the change in main as well ?

Reply:
I think changing this in the backport only makes more sense, if we consider the opposite scenario: if the code was already here in v1.16, then it could have been a worthy target of a code overhaul as part of using new 1.23+ Go features, and in this case the old code would be left alone, right?

Reply:
Pushed a change with the proposed patch, thanks!

Reply:
Ha I like that, makes sense thanks!